velox/type/ShortDecimal.h

```diff
@@ -71,8 +73,13 @@ struct ShortDecimal {
   }
 
  private:
-  const int64_t unscaledValue_;
+  int64_t unscaledValue_;
 };
+
+template <typename T>
+void toAppend(const ::facebook::velox::ShortDecimal& value, T* result) {
+  VELOX_UNSUPPORTED();
```

Comment:
add a comment as to why this cannot be supported.

Reply:
This change is part of previous commit. I will modify there.